codespell_lib/_codespell.py

```diff
@@ -1063,10 +1063,8 @@ def main(*args: str) -> int:
         return EX_USAGE
     uri_ignore_words = parse_ignore_words_option(options.uri_ignore_words_list)
 
-    if options.dictionary:
-        dictionaries = options.dictionary
-    else:
-        dictionaries = ["-"]
+    dictionaries = options.dictionary if options.dictionary else ["-"]
```

Comment:
This could be equivalently expressed more compactly like this but what you have is fine, too
```suggestion
    dictionaries = options.dictionary or ["-"]
```

Reply:
Good to know! Haven't seen that syntax before.